etc/config.xml

```diff
@@ -53,6 +53,7 @@
           </advanced_paytrail_personalization>
         </paytrail_personalization>
         <checkout_algorithm>sha256</checkout_algorithm>
+        <order_place_redirect_url>1</order_place_redirect_url>
```

Comment:
Incorrect default value, value should be 0 to be backwards compatible

Reply:
Changed to 0